pkg/api/controller.go

```diff
@@ -258,6 +258,13 @@ func (c *Controller) InitImageStore(reloadCtx context.Context) error {
 				c.Config.Storage.Commit, c.Log, c.Metrics, store)
 		}
 
+		// Enable running garbage-collect periodically
```

Comment:
Can I have the `gcInterval` disabled at rootDir level but enabled at subPath level? Then how does this change?

Reply:
The logic for gc periodic is the same as the one for gc inline. So basicaly this can be enabled/disabled per ImageStore which can be the DefaultStore or one of the SubStores. gcInterval can be disabled at rootDir level(if in config file "gc": false or "gc": true, but no value specified for "gcInterval") and in the same time can be enabled at subPath level if "gc":true and the value for "gcInterval" > 0 for one of the SubStores under "subPaths" field.